### PR TITLE
Add a page for each collection.

### PR DESCRIPTION
**Pull request by clementbiron**
The entire page is built using constant data from the `collection.json` file but the interface, as it is designed and integrated, easily allows the data schema to evolve to enrich the page.

### CODE REVIEW

File: i18n/en.yaml [hunk 1]
```diff
@@ -76,12 +76,30 @@ collections:
   jurisdiction: 
     one: Jurisdiction
     other: Jurisdictions
+  metadatas:
+    placeholder: <p>Open Terms Archive collections are designed to gather and organise documents according to a given industry, language, and jurisdiction.</p><p>This structure allows communities of interest to come together and pool their efforts.</p>
```
Comment (MattiSG) [inline i18n/en.yaml:80]:
```suggestion
    placeholder: <p>Open Terms Archive collections are designed to gather and organise terms according to a given industry, language, and jurisdiction.</p><p>This structure allows communities sharing similar interests to come together and pool their efforts.</p>
```

File: i18n/en.yaml [hunk 2]
```diff
@@ -76,12 +76,30 @@ collections:
   jurisdiction: 
     one: Jurisdiction
     other: Jurisdictions
+  metadatas:
```
Comment (MattiSG) [inline i18n/en.yaml:79]: “data” is invariable 🙂 
```suggestion
  metadata:
```

File: i18n/fr.yaml [hunk 3]
```diff
@@ -74,14 +74,32 @@ collections:
   language_various: diverses
   language_europe: Union européenne
   jurisdiction: 
-      one: Juridiction
-      other: Juridictions
+    one: Juridiction
+    other: Juridictions
+  metadatas:
+    placeholder: <p>Les collections Open Terms Archive sont conçues pour rassembler et organiser des documents selon une industrie, une langue et une juridiction données.</p><p>Cette structure permet de rassembler des communautés d'intêrets et de mutualiser leurs efforts.</p>
```
Comment (MattiSG) [inline i18n/fr.yaml:80]:
```suggestion
    placeholder: <p>Les collections Open Terms Archive sont conçues pour rassembler et organiser des termes selon une industrie, langue et juridiction données.</p><p>Cette structure permet de rassembler des communautés d'intêrets et de mutualiser leurs efforts.</p>
```

File: themes/opentermsarchive/assets/css/components/collection__metadatas.css
Comment (MattiSG) [inline themes/opentermsarchive/assets/css/components/collection__metadatas.css:1]: `metadata` is invariable, correct the file name 🙂

**Review by MattiSG — CHANGES_REQUESTED** [review]
Good!

Here are some suggestions for the rendered page:

- I feel that the header CTAs would look better if they all had the same width.
image

- I would like each section to have a subtitle or little description. They currently look rather empty and mysterious: “contribute” could read for example “Ways in which you can support the community tracking terms in this collection”; “memos” could read for example “Memos are short analyses of significant changes that clearly explain their consequences for end users.”; “governance” could read “These people and entities ensure that terms in this collection are tracked well.”

**Comment by MattiSG** [comment]
I feel that adding a secondary CTA on the homepage listing, “See details”, would make this way more relevant without needing significant work.

File: i18n/en.yaml [hunk 4]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
-    other: Languages
-  language_various: various
-  language_europe: European Union
-  jurisdiction: 
-    one: Jurisdiction
-    other: Jurisdictions
+  metadata:
+    placeholder: <p>Open Terms Archive collections are designed to gather and organise terms according to a given industry, language, and jurisdiction.</p><p>This structure allows communities sharing similar interests to come together and pool their efforts.</p>
+    services: Number of services
+    documents: Number of documents
```
Comment (MattiSG) [inline i18n/en.yaml:72]:
```suggestion
    services: Tracked services
    documents: Tracked terms
```

File: i18n/fr.yaml [hunk 5]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
+    language_europe: Union européenne
+    jurisdiction: 
       one: Juridiction
       other: Juridictions
   cta:
     browse: Voir les documents
+    details: Voir les détails
     download: Télécharger le jeu de données
-    endpoint: API de métadonnées
+    endpoint: Accès à l'API
```
Comment (MattiSG) [inline i18n/fr.yaml:85]:
```suggestion
    endpoint: Accéder à l'API
```

File: i18n/fr.yaml [hunk 6]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
   volunteer_contributors: Contributeurs bénévoles
   new: Vous voulez suivre des services visant une autre langue, juridiction ou industrie ? Contactez-nous pour commencer à collaborer !
+  contribute:
+    title: Contribuer
+    subtitle: Les façons dont vous pouvez soutenir la communauté à suivre les termes de cette collection.
```
Comment (MattiSG) [inline i18n/fr.yaml:90]:
```suggestion
    subtitle: Les façons dont vous pouvez rejoindre la communauté qui suit les termes de cette collection.
```

**Review by Ndpnt — APPROVED** [review]

File: i18n/fr.yaml [hunk 7]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
+    donate: Donner
+    publish: Publier un mémo
+    track: Suivre plus de documents
+    restore: Restaurer les documents en panne
```
Comment (MattiSG) [inline i18n/fr.yaml:94]:
```suggestion
    track: Suivre des termes supplémentaires
    restore: Réparer des termes mal suivis
```

File: i18n/fr.yaml [hunk 8]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
+    details: Voir les détails
     download: Télécharger le jeu de données
-    endpoint: API de métadonnées
+    endpoint: Accès à l'API
   volunteer_contributors: Contributeurs bénévoles
   new: Vous voulez suivre des services visant une autre langue, juridiction ou industrie ? Contactez-nous pour commencer à collaborer !
+  contribute:
+    title: Contribuer
+    subtitle: Les façons dont vous pouvez soutenir la communauté à suivre les termes de cette collection.
+    donate: Donner
```
Comment (MattiSG) [inline i18n/fr.yaml:91]:
```suggestion
    donate: Faire un don
```

File: i18n/fr.yaml [hunk 9]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
+    publish: Publier un mémo
+    track: Suivre plus de documents
+    restore: Restaurer les documents en panne
+    promote: Promouvoir
```
Comment (MattiSG) [inline i18n/fr.yaml:95]:
```suggestion
    promote: Faire connaître
```

File: i18n/en.yaml [hunk 10]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
     browse: Browse terms
+    details: View details
     download: Download dataset
-    endpoint: Metadata API
+    endpoint: Access API
   volunteer_contributors: Volunteer contributors
   new: Want to track services for different languages, jurisdictions, or industries? Contact us to collaborate!
+  contribute:
+    title: Contribute
+    subtitle: Ways in which you can support the community tracking terms in this collection.
```
Comment (MattiSG) [inline i18n/en.yaml:90]:
```suggestion
    subtitle: Ways in which you can join the community tracking terms in this collection.
```

File: i18n/fr.yaml [hunk 11]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
+  memos:
+    title: Mémos
+    subtitle: Les mémos sont des analyses courtes des changements significatifs qui expliquent clairement leurs conséquences pour les utilisateurs finaux.
```
Comment (MattiSG) [inline i18n/fr.yaml:98]:
```suggestion
    subtitle: Un mémo est une courte analyse de changements significatifs qui explique clairement leurs conséquences pour les utilisateurs.
```

File: i18n/fr.yaml [hunk 12]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
+    cta: Voir tous les mémos
+  governance:
+    title: Gouvernance
+    subtitle: Ces personnes et entités s'assurent que les termes de cette collection sont bien suivis.
+    maintainers:
+      title: Mainteneurs
+    contributors:
+      title: Contributeurs
```
Comment (MattiSG) [inline i18n/fr.yaml:106]:
```suggestion
      title: Contributeurices
```

File: i18n/fr.yaml [hunk 13]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
+    promote: Promouvoir
+  memos:
+    title: Mémos
+    subtitle: Les mémos sont des analyses courtes des changements significatifs qui expliquent clairement leurs conséquences pour les utilisateurs finaux.
+    cta: Voir tous les mémos
+  governance:
+    title: Gouvernance
+    subtitle: Ces personnes et entités s'assurent que les termes de cette collection sont bien suivis.
+    maintainers:
+      title: Mainteneurs
```
Comment (MattiSG) [inline i18n/fr.yaml:104]:
```suggestion
      title: Maintenance
```

File: i18n/fr.yaml [hunk 14]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
-  language:
-    one: Langue
-    other: Langues
-  language_various: diverses
-  language_europe: Union européenne
-  jurisdiction: 
+  metadata:
+    placeholder: <p>Les collections Open Terms Archive sont conçues pour rassembler et organiser des termes selon une industrie, langue et juridiction données.</p><p>Cette structure permet de rassembler des communautés d'intêrets et de mutualiser leurs efforts.</p>
+    services: Nombre de services
+    documents: Nombre de documents
```
Comment (MattiSG) [inline i18n/fr.yaml:72]:
```suggestion
    services: Services suivis
    documents: Termes suivis
```

File: i18n/en.yaml [hunk 15]
```diff
@@ -66,22 +66,44 @@ product_newsletter:
...
+    track: Track more terms
+    restore: Restore failing terms
+    promote: Promote
+  memos:
+    title: Memos
+    subtitle: Memos are short analyses of significant changes that clearly explain their consequences for end users.
+    cta: See all memos
+  governance:
+    title: Governance
+    subtitle: These people and entities ensure that terms in this collection are tracked well.
```
Comment (MattiSG) [inline i18n/en.yaml:102]:
```suggestion
    subtitle: These people and entities ensure that terms in this collection are tracked properly.
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
Nice!
The order in which the collections are displayed on the homepage is not the same anymore, we had worked on choosing that order, it would be good to keep it 🙂

**Comment by clementbiron** [comment]
> The order in which the collections are displayed on the homepage is not the same anymore, we had worked on choosing that order, it would be good to keep it 🙂

Fixed

**Review by MattiSG — APPROVED** [review]
🚀

**Pull request merged**